### PR TITLE
Feature/#72

### LINKED ISSUE #72: Implement SettingsAutocomplete Service
- create
- delete
- update

### PR DESCRIPTION
**Pull request by IvanKostiv**
- Added SettingAutocompleteService

### CODE REVIEW

File: src/app/services/setting_autocomplete.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+        :param setting_autocomplete_id: required param
+        :param data_url:
+        :param sheet:
+        :param from_row:
+        :param to_row:
+        :param field_id:
+        :return: updated model or None
+        """
+        setting_autocomplete = SettingAutocomplete.query.get(setting_autocomplete_id)
```
Comment (yurdosii) [inline src/app/services/setting_autocomplete.py]: Why not to use get_by_id method?

File: src/app/services/setting_autocomplete.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+from app.helper.decorators import transaction_decorator
+from app.helper.errors import SettingAutocompleteNotExist
+
+
+class SettingAutocompleteService:
+    """
+    Class for SettingAutocomplete model
+    """
+    @staticmethod
+    @transaction_decorator
```
Comment (yurdosii) [inline src/app/services/setting_autocomplete.py]: Missing new line after docstring

Comment (IvanKostiv) [reply src/app/services/setting_autocomplete.py]: Fixed

File: src/app/services/setting_autocomplete.py (hunk 1 above)
Comment (IvanKostiv) [reply src/app/services/setting_autocomplete.py]: Fixed

File: src/app/services/setting_autocomplete.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        :param sheet:
+        :param from_row:
+        :param to_row:
+        :param field_id:
+        :return: updated model or None
+        """
+        setting_autocomplete = SettingAutocompleteService.get_by_id(setting_autocomplete_id)
+
+        if setting_autocomplete is None:
+            raise SettingAutocompleteNotExist
```
Comment (yurdosii) [inline src/app/services/setting_autocomplete.py]: SettingAutocompleteNotExist -> SettingAutocompleteNotExist()

File: src/app/services/setting_autocomplete.py [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        """
+        Delete SettingAutocomplete model by id
+
+        :param setting_autocomplete_id:
+        :return: True if model is deleted or None
+        """
+        setting_autocomplete = SettingAutocompleteService.get_by_id(setting_autocomplete_id)
+
+        if setting_autocomplete is None:
+            raise SettingAutocompleteNotExist
```
Comment (yurdosii) [inline src/app/services/setting_autocomplete.py]: SettingAutocompleteNotExist -> SettingAutocompleteNotExist()

**Review by Howrran — DISMISSED** [review]

**Review by yurdosii — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

File: src/app/services/setting_autocomplete.py (hunk 3 above)
Comment (IvanKostiv) [reply src/app/services/setting_autocomplete.py]: Fixed

**Review by Howrran — APPROVED** [review]

File: src/app/services/setting_autocomplete.py (hunk 4 above)
Comment (IvanKostiv) [reply src/app/services/setting_autocomplete.py]: Fixed

**Pull request merged**